fdm-app/CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 # Changelog fdm-app
 
+## 0.13.0
+
+### Minor Changes
+
+- c5c3dd9: Do not show example values for advice and limit at fertilizer application cards
+- 3759d58: Add a card at fertilizer applications with workable nitrogen
+
```

Comment:
for next branch: add cards for N surplus, with current and desired one. we need to discuss how to adress field versus farm level for N surplus card (combined into one or separate). Just as a note here.

Reply:
thanks. well done. saves much time ;-)